### PR TITLE
Add comments and remove the duplicated PredefinedNamespace test descriptions

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
* Just add descriptions and some comments for a better understanding of the testflow

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-6042

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/e2e/specs/miscellaneous/PredefinedNamespace.spec.ts [hunk 1]
```diff
@@ -65,22 +65,23 @@ suite(`Create predefined workspace and check it ${BASE_TEST_CONSTANTS.TEST_ENVIR
...
 	// verify that just created workspace is available for the dedicated user
-	test('Validate the created workspace is present in predefined namespace', function (): void {
+	test('Validate that predefined namespace has been created', function (): void {
 		const expectedProject: string = shellExecutor.executeArbitraryShellScript('oc get projects');
 		expect(expectedProject).contains(predefinedNamespaceName);
 	});
 
 	// make sure that the generated devspace has been created in the predefined namespace
-	test('Validate the created workspace is present in predefined namespace', function (): void {
+	test('Create test DevSpace and make sure that it has been created in the predefined project', function (): void {
```
Comment (dmytro-ndp) [inline tests/e2e/specs/miscellaneous/PredefinedNamespace.spec.ts]: Test is actually waiting on workspace to be available, isn't it?
If so, `Make sure that test workspace has been created in the predefined project` test name sounds more relevant, IMHO.

File: tests/e2e/specs/miscellaneous/PredefinedNamespace.spec.ts [hunk 2]
```diff
@@ -65,22 +65,23 @@ suite(`Create predefined workspace and check it ${BASE_TEST_CONSTANTS.TEST_ENVIR
...
 		// relogin under the admin user (because regular user does not have permissions for getting pod states)
 		kubernetesCommandLineToolsExecutor.loginToOcp('admin');
 		expect(kubernetesCommandLineToolsExecutor.waitDevWorkspace().stdout).contains('condition met');
 	});
 
-	// verify that just created workspace with unique name is present in the predefined namespace
-	test('Validate the created workspace is present in predefined namespace', function (): void {
+	// verify that just created workspace with unique name is present in the predefined namespace. Potentially several users can use the same cluster
+	// and create several DevSpaces in the same OpenShift project. We need to make sure that there was created our test project by uniq name
+	test('Validate that has been created correct DevSpace with unique name)', function (): void {
```
Comment (dmytro-ndp) [inline tests/e2e/specs/miscellaneous/PredefinedNamespace.spec.ts]: did you mean **workspace**, not DevSpace?

Comment (musienko-maxim) [reply tests/e2e/specs/miscellaneous/PredefinedNamespace.spec.ts]: Change descriptions look please again

Comment (musienko-maxim) [reply tests/e2e/specs/miscellaneous/PredefinedNamespace.spec.ts]: Change the description one more time. Reread pls.

**Review by dmytro-ndp — APPROVED** [review]
looks good to merge

**Pull request merged**